benchmarks/eko/benchmark_evol_to_unity.py

```diff
@@ -104,7 +104,7 @@ def test_operator_grid(
             o_back.op_members[(21, 21)].value @ o.op_members[(21, 21)].value
             + o_back.op_members[(21, 100)].value @ o.op_members[(100, 21)].value,
             np.eye(dim[0]),
-            atol=9e-2,
+            atol=11.2e-2,
```

Comment:
Is this not too bad? You're failing of `0.112` on something that has to be `[0, 1]`...

Reply:
I checked that this large discrepancy occurs in the second-last row of the matrix so basically when you have only one off diagonal element which should be 0 but it's not...